tests/unit/test_completion.py

```
@@ -161,6 +172,143 @@ async def test_acompletion_rejects_prompt_cache_key_for_unsupported_provider() -
+                    "extra_content": {"google": {"thought_signature": "c2ln"}},
+                }
+            ],
+        )
+        assert message.tool_calls is not None
+        (tool_call,) = message.tool_calls
+        assert isinstance(tool_call, ChatCompletionMessageFunctionToolCall), type(tool_call)
+        assert tool_call.extra_content == {"google": {"thought_signature": "c2ln"}}
+        """
+    )
+
+    result = subprocess.run([sys.executable, "-c", script], capture_output=True, text=True, check=False)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)